docs/config/README.md

```diff
@@ -262,7 +262,7 @@ running `eval $(starship init $0)`, and then proceed as normal.
 | Variable   | Default         | Description                                                |
 | ---------- | --------------- | ---------------------------------------------------------- |
 | `min_time` | `2`             | Shortest duration to show time for.                        |
-| `prefix`   | `took`          | Prefix to display immediately before the command duration. |
+| `prefix`   | `took `         | Prefix to display immediately before the command duration. |
```

Comment:
Did you intend to make this change?

Reply:
The default value seems to be `took ` when I inspect the code.